pkg/ruler/ruler.go

```diff
@@ -250,6 +259,13 @@ func NewRuler(cfg Config, manager MultiTenantManager, reg prometheus.Registerer,
 		}, []string{"reason"}),
 	}
 
+	if len(cfg.EnabledTenants) > 0 {
+		level.Info(ruler.logger).Log("msg", "using enabled users", "enabled", strings.Join(cfg.EnabledTenants, ", "))
```

Comment:
[nit] in single-binary mode may not be cristal clear who is logging it:

```suggestion
		level.Info(ruler.logger).Log("msg", "ruler is using enabled users", "enabled", strings.Join(cfg.EnabledTenants, ", "))
```

Reply:
It should be visible from `caller`, but let's fix the messages too. 👍